string label #Class of the object detected
float64 confidence #Probability associated with the object [0-1]
vision_msgs/BoundingBox2D bbox #Bounding box of the object
sensor_msgs/Image mask #Segmentation Mask of the object
geometry_msgs/Pose pose #Pose of the center of the object
